src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -405,6 +422,15 @@ private void setup(OptionSet options, Properties properties) {
 		this.redisListKey		= fetchOption("redis_list_key", options, properties, "maxwell");
 		this.redisType			= fetchOption("redis_type", options, properties, "pubsub");
 
+		this.natsUrl = fetchOption("nats_url", options, properties, "nats://localhost:4222");
+		this.natsClusterId = fetchOption("nats_cluster_id", options, properties, "test-cluster");
+		this.natsClientId = fetchOption("nats_client_id", options, properties, "test-client");
+		this.natsSubject = fetchOption("nats_subject", options, properties, "test-subject");
```

Comment:
sorry, just got back to this PR.  Putting a default here of `test-subject` seems a touch optimistic, I think probably better to have it required?  I guess a default of `maxwell` wouldn't be terrible either.